### DISCUSSION TITLE
Should we validate the URL value in Dashboard settings Links section ?

### DISCUSSION THREAD

**Discussion by krishnemd** [grafana/grafana#72424] [Dashboards, Panels & Visualizations]
Any random value can be assigned as URL at the moment. URLs can be validated by simple JS function. Should we validate the URL field from UI this way and show an alert ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82540. Feel free to continue the discussion around this there. Thank you!
